### ISSUE TITLE
Move comment hash JS to Timber js file

### ISSUE THREAD

**Issue by cshold** [Shopify/Timber#273]
Remove the inline JS that changes the hash when a comment is posted and move it to an external sheet.

As per @mpiotrowicz and @stevebosworth's suggestion:

>I'd argue strongly that JS should be kept in JS files. Ideally, inline JS should be limited to setting liquid->dependent JS vars or calling functions

>eg this would call

>timber.updateHash(#comments)

>In this specific case as well, one thing to consider is adding in nicer keyboard support to these events too. For example right now if that page were to load with a form error, you'd be presented back to the form, but on hitting "tab", you'd be brought back to the top of the page, cycling through the main nav. By moving the JS inside the file, you'd be able to write a more robust function that also moved focus on the page to where that # is as well
